Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -473,17 +475,17 @@ class BrowserViewController: UIViewController,
         }
     }
 
-    func newState(state: FakespotState) {
+    func newState(state: BrowserViewControllerState) {
```

Comment:
FYI with the forthcoming iPad multi-window feature we'll have multiple BVCs running simultaneously in the same app. I think some of this will need to be updated to work with multiple windows, similar to what is being hashed out currently for the `TabManager`. 

For example any browser-related actions will probably need to include metadata about which browser instance is being acted on. And the related state will probably need some kind of UUID so it can be associated with its parent window/scene.

Much of that is still in the planning stages, but I just wanted to mention it since it will probably impact any new Redux work for BVC etc. (cc @yoanarios @OrlaM)